### ISSUE TITLE
Component request: Heat map

### ISSUE THREAD

**Issue by LoicUV** [FormidableLabs/victory#262]
Hello! As I need a [heat map](http://jsfiddle.net/gh/get/jquery/1.9.1/highslide-software/highcharts.com/tree/master/samples/highcharts/demo/heatmap/) type of chart for a project, I was wondering if some work had already been done regarding this, and if not try to implement it.

We can discuss the requirements here.

Here's a concrete example of how it would look in my project:
![heatmap](https://cloud.githubusercontent.com/assets/592542/15143347/85087492-16a9-11e6-9924-df49d8a98afa.png)

**Comment by SamEureka** [comment]
adding a 👍  to this component request

**Comment by dandelany** [comment]
If it's useful, I recently implemented a color heat map in my own library, feel free to borrow code:

https://github.com/spotify/reactochart/blob/0.2.1/src/ColorHeatmap.js

**Comment by boygirl** [comment]
Super useful. Thanks @dandelany

**Comment by petekp** [comment]
+1 for a heatmap component 😃

**Comment by karimkawambwa** [comment]
+1 👍

**Comment by camilossantos2809** [comment]
any news about that?

**Comment by boygirl** [comment]
Yes! this is the next big thing I'm planning to work on. It will build on the histogram feature we released recently

**Comment by boygirl** [comment]
**Notes:** This component will be basically a 2d histogram, and this work will involve updates / improvements to `VictortyHistogram`. Work will also involve enhancement of the `colorScale` prop or the addition of a new `colorBy` prop to support heatmap coloring. Work is in progress, but interested parties should reach out to contribute. Especially folks with API opinions.

https://github.com/FormidableLabs/victory/tree/feature/heatmap

**Comment by ofrasergreen** [comment]
Any updates on this? I'd really like to use a heatmap.

**Comment by h-kataria** [comment]
@becca-bailey I see you guys have an initial implementation of heatmaps in victory charts but didn't get that over the finish line. We would like to have them included as part of the victory charts so we can use them. Is there a way we could help you guys to get this task finished?
